pkg/controllers/vdb/onlineupgrade_reconciler.go

```diff
@@ -168,7 +171,10 @@ func (o *OnlineUpgradeReconciler) precomputeStatusMsgs(ctx context.Context) (ctr
 	if res, err := o.iterateSubclusterType(ctx, vapi.SecondarySubcluster, procFunc); verrors.IsReconcileAborted(res, err) {
 		return res, err
 	}
-	o.StatusMsgs = append(o.StatusMsgs, "Destroying transient secondary subcluster")
+	o.StatusMsgs = append(o.StatusMsgs,
+		"Reinstalling default packages",
+		"Destroying transient secondary subcluster",
```

Comment:
The order needs to flip now that we do the install after deleting the transient subcluster.

Reply:
will do